public/css/nexthome.css

```diff
@@ -492,7 +492,7 @@ body {
     .top-nav .logo a {
         width: 90px !important;
     }
-    .col-md-6 {
+    .nh-col-md-6 {
         flex: 0 0 50%;
         max-width: 100%;
     }
```

Comment:
Is it "bad" that we're still changing `.col-sm-4` and other bootstrap stuff?

Reply:
Sure. Lesson learnt.

Reply:
OK, I'm not going to change all the others here.  But will ref issue https://github.com/decred/dcrdata/issues/782,  noting that there is still more to fix.